app/services/step_by_step_page_reverter.rb

```diff
@@ -12,6 +12,7 @@ def repopulate_from_publishing_api
     step_by_step_page.introduction = introduction
     step_by_step_page.description = payload_from_publishing_api[:description]
     step_by_step_page.draft_updated_at = step_by_step_page.published_at
+    step_by_step_page.status = "published"
```

Comment:
Can all of these updates be replaced with a single call to `update`?

Reply:
Isn't that what happens when `save!` is called?

Reply:
I mean, could the entire methods be rewritten like this?

```ruby 
def repopulate_from_publishing_api
  update(
    title: payload_from_publishing_api[:title],
    slug: payload_from_publishing_api[:base_path].tr('/', ''),
    introduction: introduction,
    description: payload_from_publishing_api[:description],
    draft_updated_at: step_by_step_page.published_at,
    status: "published"
  )
end
```

Reply:
Update and fixed-up